runtime/tests/runtime_integration_tests/test_set_auth_manifest.rs

```diff
@@ -0,0 +1,128 @@
+// Licensed under the Apache-2.0 license
+
+use crate::common::run_rt_test;
+use caliptra_auth_man_gen::{
+    AuthManifestGenerator, AuthManifestGeneratorConfig, AuthManifestGeneratorKeyConfig,
+};
+use caliptra_auth_man_types::{
+    AuthManifestImageMetadata, AuthManifestPrivKeys, AuthManifestPubKeys,
+    AUTH_MANIFEST_VENDOR_SIGNATURE_REQURIED_FLAG,
+};
+use caliptra_common::mailbox_api::{CommandId, MailboxReq, MailboxReqHeader, SetAuthManifestReq};
+use caliptra_hw_model::HwModel;
+use caliptra_image_crypto::OsslCrypto as Crypto;
+use caliptra_image_fake_keys::*;
+use caliptra_runtime::RtBootStatus;
+use zerocopy::AsBytes;
+
+#[test]
```

Comment:
Is there a plan to add a test for each of the possible errors above? (mismatched signatures, mismatched marker, size bounds, etc.)